--- PKGBUILD.orig	2018-06-27 17:20:32.000000000 -0500
+++ PKGBUILD	2018-06-27 17:22:44.000000000 -0500
@@ -82,6 +82,7 @@
   # Compile fixes.
   "ocaml-unsafe-string.patch"
   "qemu-xen-memfd.patch::https://github.com/qemu/qemu/commit/75e5b70e6b5dcc4f2219992d7cffa462aa406af0.patch"
+  "tools_libacpi_Makefile.patch"
   "xen-gcc-8-libxc-fix-strncpy-size.patch::https://xenbits.xen.org/gitweb/?p=xen.git;a=patch;h=fa7789ef18bd2e716997937af71b2e4b5b00a159"
   "xen-gcc-8-misc-fix-hypothetical-buffer-overflow-in-xen-lowmemd.patch::https://xenbits.xen.org/gitweb/?p=xen.git;a=patch;h=27751d89248c8c5eef6d8b56eb8f7d2084145080"
   "xen-gcc-8-fix-possible-null-truncation.patch::https://xenbits.xen.org/gitweb/?p=xen.git;a=patch;h=938c8f53b1f80175c6f7a1399efdb984abb0cb8b"
@@ -115,6 +116,7 @@
   # Compile fixes.
   "7c76b116ce09a53708306682f04e1460a788fe66f832091b7003a5d8e1fee312"
   "29004b3b9f79bb2cdb0553c5a77c8d748a92e628405b7d9f9ae46693515757bb"
+  "2397bc57ac315b18873276df790d2d9c10c51cb324ea3abd39e30ef39ea0cb5a"
   "fc1a4ce85795a9a00161d5188b61db5054c462db9a65a8073a5b55b49b089e4d"
   "3fefa746363a0b05e4b90b18f4a510cd1eb10cd4ea2b845fa6099a5622490724"
   "4e5a7c72ed9b40c9f1f47d2dce3d4501d293bec208cd170ff5702cf9a1fc3a01"
@@ -196,6 +198,7 @@
   # Misc compile fixes (removed in future versions if not needed anymore).
   msg2 'Applying misc compile fixes...'
   patch -Np1 -i "${srcdir}/ocaml-unsafe-string.patch"
+  patch -Np1 -i "${srcdir}/tools_libacpi_Makefile.patch"
   patch -Np1 -i "${srcdir}/xen-gcc-8-libxc-fix-strncpy-size.patch"
   patch -Np1 -i "${srcdir}/xen-gcc-8-misc-fix-hypothetical-buffer-overflow-in-xen-lowmemd.patch"
   patch -Np1 -i "${srcdir}/xen-gcc-8-fix-possible-null-truncation.patch"
