Avoid creating and chmod'ing $srcdir unnecessarily

--- makepkg.orig	2017-12-18 17:27:49.000000000 -0600
+++ makepkg	2017-12-18 17:29:38.000000000 -0600
@@ -2344,8 +2344,12 @@
 fi
 
 # get back to our src directory so we can begin with sources
-mkdir -p "$srcdir"
-chmod a-s "$srcdir"
+if [[ ! -d "$srcdir" ]]; then
+	mkdir -p "$srcdir"
+fi
+if [[ -k "$srcdir" ]]; then
+	chmod a-s "$srcdir"
+fi
 cd_safe "$srcdir"
 
 if (( !REPKG )); then
